### ISSUE TITLE
Range queries on secondary indexes

### ISSUE THREAD

**Issue by plentylife** [ClickSimply/Nano-SQLite3#1]
Hi Scott,

It appears (though I am not certain) that this issue https://github.com/ClickSimply/Nano-SQL/issues/42 exists when using nano-sqlite.

I'm sorry I can't spend more time to confirm. I'm going away for the summer soon, so it's busy.

Best wishes,
Anton

**Comment by ClickSimply** [comment]
Hey Anton,

The secondary index query code exists only in NanoSQL, it's likely that you're actually using an older version of nanoSQL.  Take a look in `node_modules/nano-sqlite/node_modules` and see if there's an older version of nanoSQL in there.  If so, that's your problem. 👍 

Two solutions:
1. Delete the `node_modules/nano-sqlite/node_modules/nano-sql` folder so the main nano-sql in your node_modules folder is used.
2. I just updated nano-sqlite to require the newest version of nanoSQL so doing `npm i nano-sqlite@latest` should also resolve the issue.

**Comment by plentylife** [comment]
Hm...
I did check for that. There wasn't a `nano-sql` package inside `node_modules`.

It is quite possibly a bug in my code of some sorts. 
I switched to `leveldb`, which works now and didn't used to (I can't remember if I couldn't get the leveldown package to install or a database viewer)

If I ever get my hands around it, I'll test it again, but for now I just can't.

I'll close this issue, but perhaps if it's real someone else will stumble upon it.

Thank you.
